commands/releases/status_helper.js

```diff
@@ -1,25 +1,35 @@
 'use strict'
 
-module.exports = function (s) {
+function pendingDescription (release, runningRelease, runningSlug) {
+  if (
```

Comment:
We'd still need to check that `runningRelease.id == release.id`, but removing the `!== undefined` does simplify it.